### ISSUE TITLE
Rust: Auto-completion doesn't show full path

### ISSUE THREAD

**Issue by DragonDev1906** [zed-industries/zed#14767]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

When using `editor::ShowCompletions` (`ctrl-space`) in a Rust project on a common name it does not show the full path of the identifiers.

I often use this to select from where I want to import something (my project has over 20 `Config` symbols for example and I don't want to know how many `Error` symbols most projects have). Unfortunately that is really hard when the resulting suggestion list does not contain information where the suggested symbol comes from.

**Example:**
```rust
mod A {
    pub struct Config;
}
mod B {
    pub struct Config;
}
mod C {
    pub struct Config;
}

// No imports yet

fn main() {
    let x = Config{} // Attempt auto-completion on `Config`
}
```

The resulting auto-completion suggestions show 3x `Config` but doesn't tell which one is which. Therefore you effectively cannot use auto-completion suggestions to add a missing import.

It would be amazing if the auto-completion list would show the following. I still only want `Config` in the code.
```
Config (use A::Config)
Config (use B::Config)
Config (use C::Config)
```

Alternatively (though it's a lot less useful when trying to find the correct one) this information could be shown in the docstring panel that is shown. The main downside to that is that you have to go through all suggestions to find the one you're looking for.

Not sure if I should post this as a bug issue or a feature request. I'd consider it a bug because it effectively makes auto-completion useless for the situation where you know the exact name and have to select which one you want to import.

### Environment

Zed: v0.143.7 (Zed)
OS: Linux X11 arch unknown
Memory: 31 GiB
Architecture: x86_64

### If applicable, attach your Zed.log file to this issue.

**Comment by osiewicz** [comment]
Hey, that's been fixed in the most recent Preview (0.145.0); give it a go and let me know how it goes.

**Comment by DragonDev1906** [comment]
Sometimes it helps reading the preview Changelog. :smile:
It works as expected and makes this a lot more useful, thank you.
